cmd/bundlectl/integration-tests/BUILD

```diff
@@ -0,0 +1,9 @@
+sh_test(
```

Comment:
Can we use the directory `integration_tests` or `integration`? `blaze` (internal) doesn't play nice with foo-bar directories, as I recall